Assignment 5.Rmd

```diff
@@ -66,7 +67,7 @@ Upload the data "intelligent_tutor_new.csv". This is a data set of a differnt sa
 ```{r}
```

Comment:
Do you have any advice for the teacher based on your model? (question on line 65)